tests/core/test_migrate.py

```diff
@@ -916,3 +941,22 @@ def test_migration_cleanup(tmp_path: Path,):
     )
     assert not Path.exists(current_dir / "original_domain")
     assert not Path.exists(current_dir / "new_domain")
+
+
+def test_migration_stops_when_migrated_files_are_found(tmp_path: Path,):
+    domain_dir = tmp_path / "domain"
+    domain_dir.mkdir()
+    prepare_domain_path(
+        domain_dir,
+        """
+        version: "3.0"
+        """,
+        "domain.yml",
+    )
+
+    with pytest.raises(
+        RasaException,
+        match=f"The domain directory '{domain_dir}' does not contain any domain "
```

Comment:
A bit confused here why this exception message gets raised? Shouldn't the message be the one that says that `some of the given files were already migrated to 3.0`?

Reply:
Ohhh, yep, forgot to fix this. Files not containing any of the indicator keys (like "intents" and so on) are not considered domain files. So the above is no domain file actually -- Need to add some key and change the message here

Reply:
done